ethereum/referencetests/build.gradle

```diff
@@ -55,7 +55,7 @@ task ('validateReferenceTestSubmodule') {
     def submoduleHashes = [
       [
         '2fe00d76d97e35aa2df6449b5f567c41cf6f58fc',
```

Comment:
Why is this the qbft reference test submodule hash even being checked as part of the ethereum reference tests module? Don't think this should be here at all. Can we change the command to only look for the ethereum/referencetests git submodule?

Reply:
The change that added the QBFT reference tests undid the check for the ethereum reference tests.  We need to make sure that the reference tests are not accidentally changed via git merges.  Which happens very frequently after updates (one which just happened)

Reply:
Perhaps we should update line 66 as well and add a third "human readable" description of the sub modeile, as well as line 69.  This should improve clarity.

Reply:
I've just pushed another fix ... each submodule respectively checks its own hash.

Reply:
That's a nicer fix